# originated from https://git.alpinelinux.org/aports/plain/main/gnupg/0410-make-gpg-zip-use-tar-from-path.patch
From: Daniel Kahn Gillmor <dkg@fifthhorseman.net>
Date: Sun, 18 Nov 2018 17:29:52 -0500
Subject: Make gpg-zip use tar from $PATH

Apparently there is no clean way to configure this from ./configure,
and upstream is deprecating gpg-zip anyway.  So just force-set tar to
be manually "tar" (meaning, that we should look in the $PATH at
runtime).

See also https://dev.gnupg.org/T4251 and https://bugs.debian.org/913582
---
 tools/gpg-zip.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Patch-Source: https://sources.debian.org/data/main/g/gnupg2/2.2.27-2/debian/patches/Make-gpg-zip-use-tar-from-PATH.patch

diff --git a/tools/gpg-zip.in b/tools/gpg-zip.in
index 9047e36..3821f3a 100644
--- a/tools/gpg-zip.in
+++ b/tools/gpg-zip.in
@@ -23,7 +23,7 @@
 # the GNU or POSIX variant of USTAR.
 
 VERSION=@VERSION@
-TAR=@TAR@
+TAR=tar
 GPG=gpg
 
 usage="\
